src/components/widgets/SelectWidget.js

```diff
@@ -23,15 +23,30 @@ function SelectWidget({
   value,
   defaultValue,
   required,
+  multiple,
   onChange
 }) {
   return (
     <select
+      multiple={multiple}
```

Comment:
No, when you use `multiple`, it means `multiple={ true }` (it is not the same as in objects)